### PR TITLE
Formatted the README.md

### PR DESCRIPTION
**Pull request by ojcorrigan**
I've added some formatting to try and make the readme look nicer. 

ticket #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,74 @@
-BACKEND PORTFOLIO PROJECT 
+# BACKEND PORTFOLIO PROJECT
 
-Welcome to my very first app. I have created it as part of the Northcoders bootcamp. 
+<p>Welcome to my very first app. I have created it as part of the Northcoders bootcamp.</p>
```
Comment (jadelandeg) [inline README.md]: don't want to talk in the first person here. Just talk about what the app is and what it does. This app was created as part of Northcoders bootcamp. It has several endpoints etc etc

File: README.md [hunk 2]
```diff
@@ -1,34 +1,74 @@
...
 
-I have made a simple app with several endpoints (all of which can be viewed in the endpoints.json file.) The aim was to create a news app which could provide the client with information about news articles, comments, users and topics. It also allows for posting of new comments, deletion of comments by id and the patching of articles. 
+<p>I have made a simple app with several endpoints (all of which can be viewed in the endpoints.json file.)<br> 
+The aim was to create a news app which could provide the client with information about news articles, comments, users and topics.<br> It also allows for posting of new comments, deletion of comments by id and the patching of articles.</p>
```
Comment (jadelandeg) [inline README.md:6]: also talk about what technologies you used to build it

File: README.md [hunk 3]
```diff
@@ -1,34 +1,74 @@
...
+
+You will require a **.env.test** and a **.env.development** file to be able to connect to the appropriate database. </p>
+
+<p>to seed the databases you will need to run the command
+
+`npm run setup-dbs `
+
+this will allow the app to run as intended.</p>
+
+---
```
Comment (jadelandeg) [inline README.md:70]: don't forget about seeding the database

**Review by jadelandeg — CHANGES_REQUESTED** [review]
Looking good! Just a few changes

File: README.md (hunk 3 above)
Comment (karlaferry) [reply README.md:70]: Adding to what Jade said, after running setup-dbs you also want to run `npm run seed` and `npm run start`.

**Review by karlaferry — CHANGES_REQUESTED** [review]
Looks AWESOME Owen! Just one final comment about the scripts you need to run for this API to work (I added it to one of Jade's comments) then you're done 😄

**Pull request merged**